src/sass_context.cpp

```diff
@@ -375,6 +375,11 @@ extern "C" {
     return options;
   }
 
+  const char* ADDCALL sass_copy_c_string (const char* input_string)
```

Comment:
Not sure if I follow? If you request a copy it's your reponsability to free it!?